source/common/redis/command_splitter_impl.h

```diff
@@ -34,72 +34,67 @@ class CommandHandlerBase {
   ConnPool::Instance& conn_pool_;
 };
 
-class AllParamsToOneServerCommandHandler : public CommandHandler,
-                                           CommandHandlerBase,
-                                           Logger::Loggable<Logger::Id::redis> {
+/**
+ * SimpleRequest hashes the first argument as the key and sends the request to a single Redis.
+ */
+class SimpleRequest : public SplitRequest, public ConnPool::PoolCallbacks {
 public:
-  AllParamsToOneServerCommandHandler(ConnPool::Instance& conn_pool)
-      : CommandHandlerBase(conn_pool) {}
+  static SplitRequestPtr create(ConnPool::Instance& conn_pool, const RespValue& incoming_request,
+                                SplitCallbacks& callbacks);
 
-  // Redis::CommandSplitter::CommandHandler
-  SplitRequestPtr startRequest(const RespValue& request, SplitCallbacks& callbacks) override;
+  void onResponse(RespValuePtr&& response);
 
-private:
-  struct SplitRequestImpl : public SplitRequest, public ConnPool::PoolCallbacks {
-    SplitRequestImpl(SplitCallbacks& callbacks) : callbacks_(callbacks) {}
-    ~SplitRequestImpl();
+  void onFailure();
```

Comment:
All overrides should have "override" postfix, del newline between definitions, and add // Redis::ConnPool::PoolCallbacks between them. Same in other places.